MAUI/DataGrid/styling.md

```diff
@@ -470,4 +470,32 @@ public partial class MainPage : ContentPage
 {% endhighlight %}
 {% endtabs %}
 
-![Changing the border width in .NET MAUI DataGrid](Images/styling/maui-datagrid-border-width.png)
\ No newline at end of file
+![Changing the border width in .NET MAUI DataGrid](Images/styling/maui-datagrid-border-width.png)
+
+## Changing the border color.
+
+You can change the border color by using [GridLineColor](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataGrid.DataGridStyle.html#Syncfusion_Maui_DataGrid_DataGridStyle_GridLineColorProperty) property in `SfDataGrid.DefaultStyle`.
```

Comment:
## Change the border color

The grid line color can be customized by setting the `DataGridStyle.GridLineColor` property.

Reply:
Refactor the code